packages/terra-dialog-modal/tests/wdio/dialog-modal-spec.js

```diff
@@ -0,0 +1,46 @@
+/* global browser, Terra */
+const selector = '#root';
+
+Terra.viewports('tiny', 'medium', 'large').forEach((viewport) => {
+  describe('Default Dialog Modal', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/terra-dialog-modal/dialog-modal/default-dialog-modal');
+      browser.setViewportSize(viewport);
```

Comment:
Set the viewport only once for each of tiny, medium, and large in a before block outside of each implementation to improve test performance.